.babelrc

```diff
@@ -15,5 +15,5 @@
     }],
     "@babel/preset-react"
   ],
-  "plugins": ["@babel/plugin-proposal-object-rest-spread", "syntax-async-functions", "@babel/plugin-transform-regenerator", "flow-react-proptypes", "@babel/transform-flow-strip-types", "@babel/plugin-syntax-dynamic-import", "@babel/plugin-proposal-class-properties", "@babel/plugin-proposal-private-methods", ["babel-plugin-webpack-alias-7", {"config": "./webpack.config.base.js"}], "css-modules-transform", "@babel/plugin-proposal-export-default-from", "@babel/plugin-transform-modules-commonjs", "@babel/plugin-proposal-optional-chaining"]
+  "plugins": ["@babel/plugin-proposal-object-rest-spread", "syntax-async-functions", "@babel/plugin-transform-runtime", "@babel/plugin-transform-regenerator", "flow-react-proptypes", "@babel/transform-flow-strip-types", "@babel/plugin-syntax-dynamic-import", "@babel/plugin-proposal-class-properties", "@babel/plugin-proposal-private-methods", ["babel-plugin-webpack-alias-7", {"config": "./webpack.config.base.js"}], "css-modules-transform", "@babel/plugin-proposal-export-default-from", "@babel/plugin-transform-modules-commonjs", "@babel/plugin-proposal-optional-chaining"]
```

Comment:
why do we need this plugin?

Reply:
@babel/plugin-transform-runtime is needed to run async functions in tests
```
Automatically requires @babel/runtime/regenerator when you use generators/async functions (toggleable with the regenerator option).
Can use core-js for helpers if necessary instead of assuming it will be polyfilled by the user (toggleable with the corejs option)
Automatically removes the inline Babel helpers and uses the module @babel/runtime/helpers instead (toggleable with the helpers option).
```